src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/ServerSentEventsTransport.cs

```diff
@@ -148,7 +152,7 @@ private async Task ProcessEventStream(HttpResponseMessage response, Cancellation
                 {
                     while (true)
                     {
-                        var result = await reader.ReadAsync();
+                        var result = await reader.ReadAsync(cancellationToken);
```

Comment:
This one is tricky, we explicitly don't use the CancellationToken here because we're using CancelPendingRead to avoid exceptions (for pipe operations)